### ISSUE TITLE
Invalid .mid when using percmap

### ISSUE THREAD

**Issue by leocaseiro** [paulrosen/abcjs#884]
Hi there,

I am trying to download the MIDI that is generated by abcj, but it's invalid.
I can't play on VLC, and when I open the midi file on MuseScore, the file is empty.

PS: The exactly same notation works fine if I remove the MIDI or percmap;

Some examples of failed midis are:

```
X:1
T:percmap midi fail 1
%%MIDI drummap F 36 %bass drum 1
%%MIDI drummap c 38 %acoustic snare
%%MIDI drummap g 42 %closed hi hat
Q:1/4=50
K:C perc
[gF] g [gc] g [gF] g [gc] g | c c c c
```

or

```
X:1
T:percmap mid fail 2
%%percmap F  acoustic-bass-drum
%%percmap c  acoustic-snare
%%percmap g  closed-hi-hat
Q:1/4=50
K:C perc
[gF] g [gc] g [gF] g [gc] g | c c c c
```

I've tested the mid files by download from https://editor.drawthedots.com/

**Comment by paulrosen** [comment]
will be fixed in version 6.1.9

**Comment by leocaseiro** [comment]
Amazing! it's working here https://codesandbox.io/s/abcjs-drum-download-midi-file-that-works-on-musescore4-yg34sp

Thanks
